From e156ebfe85585b5829d7330114da8a3f2ac3d009 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 30 Nov 2020 01:17:30 +0200
Subject: [PATCH] [SW-0] boot_if: include preboot.h for u32

u32 is defined in preboot.h, so include it to have the types resolved
correctly and not depend on the user to resolve the includes order
properly.

Change-Id: I39011f627696f474797d6421549fb385d457f278
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/include/boot_if.h b/include/boot_if.h
index 52bb58b..1e12747 100644
--- a/include/boot_if.h
+++ b/include/boot_if.h
@@ -8,6 +8,7 @@
 #define _HL_BOOT_IF_H_
 
 #include <hl_boot_if.h>
+#include <preboot.h>
 
 enum kmd_msg cpu_boot_cmd_get(void);
 void cpu_boot_cmd_clr_before_wfe(void);
